webhook_to_fedora_messaging/cli.py

```diff
@@ -0,0 +1,45 @@
+import logging
+import os
+import sys
+from asyncio import run
+
+import click
+
+from webhook_to_fedora_messaging import __version__
+from webhook_to_fedora_messaging.config import setup_config, setup_database_manager
+from webhook_to_fedora_messaging.database import setup_database
+from webhook_to_fedora_messaging.main import start_service
+
+
+logger = logging.getLogger(__name__)
+
+
+@click.group()
+@click.option(
+    "-c",
+    "--conf",
+    "conf",
+    type=click.Path(exists=True),
+    help="Read configuration from the specified module",
+    default=None,
+)
+@click.version_option(version=__version__, prog_name="w2fm")
+def main(conf=None):
+    if conf:
+        os.environ["W2FM_CONFIG"] = os.path.abspath(conf)
+    try:
+        setup_config()
+        setup_database_manager()
+    except FileNotFoundError:
+        logger.error("Configuration file was not found")
+        sys.exit(1)
```

Comment:
You can also raise a `click.Abort` exception as [documented here](https://click.palletsprojects.com/en/8.1.x/exceptions/#which-exceptions-exist). The good thing about it is that it'll log things on standard error and exit with an error code by itself. Also, it'll make this block easier to test in unit tests as the click runner can catch it.